resources/extruders/seckit_skgo_extruder_0.def.json

```
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "SecKit SK-Go",
```

This should refer to the filename of the printer definition.
```suggestion
        "machine": "seckit_skgo",
```